app/assets/stylesheets/application.scss

```diff
@@ -13,29 +13,46 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
- *= depend_on_asset "Landing_Page.jpg"
  */
 
  body {
-  background-image: url("<%= asset_path('Landing_Page.jpg') %>");
+  background-image: image_url("Landing_Page.jpg");
   background-size: cover;
   background-repeat: no-repeat;
   background-position: center top;
-  /*overflow-x: hidden;*/
+  background-position: 0px 45px;
  }
- 
-/*
-.footer img {
-  float: left;
-}*/
+
+.container {
+  padding: 10px;
+  background:rgba(255,255,255,0.5);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space